tests/unit_tests/test_filters.py

```diff
@@ -269,3 +269,20 @@ def test_energyfunc():
     np.testing.assert_allclose(f.energy, new_f.energy)
     np.testing.assert_allclose(f.y, new_f.y)
     assert f.interpolation == new_f.interpolation
+
+
+def test_tabular_from_energyfilter():
+    efilter = openmc.EnergyFilter([0.0, 10.0, 20.0, 25.0])
+    tab = efilter.get_tabular(values=[5, 10, 10])
```

Comment:
I think we might need to make these floats due to the new ```/=``` operation
```suggestion
    tab = efilter.get_tabular(values=[5., 10., 10.])
```

Reply:
Good point. In fact to be safe, I'd recommend using `np.array(values, dtype=float)` in the method implementation so the onus is not on the user